package.json

```diff
@@ -21,12 +21,16 @@
     "lint": "eslint ./src --ext .js,.jsx,.ts,.tsx",
     "format": "prettier --write .",
     "format:check": "prettier --check .",
-    "generate-error-catalog": "node scripts/error-catalog-sdk/generate_error_catalog.mjs"
+    "generate-error-catalog": "node scripts/error-catalog-sdk/generate_error_catalog.mjs",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook",
+    "chromatic": "chromatic"
   },
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
     "@patternfly/patternfly": "4.219.2",
+    "@patternfly/react-charts": "6.94.7",
```

Comment:
Besides all the storybook and chromatic related dependencies, I had to add this one too.
It's a peer dependency of `app-services-ui-components` and without it I was having issues while rendering our components within storybook.